Makefile

```diff
@@ -7,25 +7,56 @@ DIST_ROOT=dist
 DIST_FOLDER_NAME=mattermost-load-test
 DIST_PATH=$(DIST_ROOT)/$(DIST_FOLDER_NAME)
 
+# GOOS/GOARCH of the build host, used to determine whether we're cross-compiling or not
+BUILDER_GOOS_GOARCH="$(shell $(GO) env GOOS)_$(shell $(GO) env GOARCH)"
+
 all: install
 
 vendor:
 	glide install
 
-install: vendor
-	$(GO) install ./cmd/loadtest
-	$(GO) install ./cmd/ltops
-	$(GO) install ./cmd/ltparse
+build-linux: vendor
+	@echo Build Linux amd64
+	env GOOS=linux GOARCH=amd64 $(GO) install -i $(GOFLAGS) $(GO_LINKER_FLAGS) ./...
+
+build-osx: vendor
+	@echo Build OSX amd64
+	env GOOS=darwin GOARCH=amd64 $(GO) install -i $(GOFLAGS) $(GO_LINKER_FLAGS) ./...
+
+build-windows: vendor
+	@echo Build Windows amd64
+	env GOOS=windows GOARCH=amd64 $(GO) install -i $(GOFLAGS) $(GO_LINKER_FLAGS) ./...
 
-package: install
+build: build-linux build-windows build-osx
```

Comment:
Can we have a target that just builds your platform? I don't really want to wait around for windows and osx to build when I just need a linux package.

Reply:
That's precisely what the the updated `install` target does (also the default target through `all`), and if you're deploying `make package` also only installs linux.

Do you think we should call it something new?

Reply:
Nope I am just blind.

Reply:
👓 :)